.mergify.yml

```
@@ -19,19 +17,9 @@ queue_rules:
           - and: # breakage fails like we thought
               - check-failure=breakage
               - label=proto:expect-breakage
-    merge_conditions:
-      - base=master
-      # Require integration tests before merging only
-      - or:
-          - label=bypass:integration
-          - check-success=integration-test-result
-  - name: main
-    commit_message_template: |-
-      {{ title }} (#{{ number }})
-
-      {{ body | trim }}
-    queue_conditions:
+    high_priority_queue_checks: &high-priority-queue-checks
       - base=master
+      - label=priority:high
```

Thinking we should be explicit this is an extension of the queue_checks:

```suggestion
    high_priority_queue_checks: &high-priority-queue-checks
      <<: *queue-checks
      - label=priority:high
```

But then I'm wondering if we actually need the `label=` condition (and `base=` condition) on the `queue_conditions` since we already have them in the `pull_request_rules` conditions?

---

this syntax doesnt work as expected. the `<<:` symbol cant be used to append/extend arrays
so 
```
shared:
  array:
    - 1
    - 2

extended_array:
  <<: *array
  - 3
```

would result in:
```
extended_array: [
  [
    1,
    2,
  ],
  3,
] 
```

rather than:
```
extended_array: [
  1,
  2,
  3,
] 
```

---

Right sorry I should have written the following according to the [docs](https://docs.mergify.com/configuration/sharing/):

 ```
    high_priority_queue_checks: &high-priority-queue-checks
      - and: *queue-checks
      - label=priority:high
```

Anyway no longer needed for this case, but I think we could still use this to avoid duplicating pull request rules conditions.